examples/employee-app/src/main/java/com/example/helidon/employee/EmployeeRepository.java

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2019 Oracle and/or its affiliates. All rights reserved. */

```

Comment:
This should use the proper copyright header.

Reply:
This should be fixed now.